### ISSUE TITLE
Make BFI less prominent in a publisher profile

### ISSUE THREAD

**Issue by Daniel-Mietchen** [fnielsen/scholia#459]
Some [feedback](https://en.wikipedia.org/w/index.php?title=User_talk:Daniel_Mietchen&oldid=855687175#Scholia?) from using Template:Scholia.

Example: https://tools.wmflabs.org/scholia/publisher/Q1619253 .
Ideally, this would move into the realm of customization once we get there.

**Comment by egonw** [comment]
And at least mention what BFI is in the first place...

![image](https://user-images.githubusercontent.com/26721/107882863-f8794880-6eeb-11eb-98b9-af77ba2e5110.png)

**Comment by carlinmack** [comment]
Is the Danish Bibliometric Research Indicator notable? https://ufm.dk/en/research-and-innovation/statistics-and-analyses/bibliometric-research-indicator

**Comment by Daniel-Mietchen** [comment]
Yes: https://da.wikipedia.org/wiki/Den_Bibliometriske_Forskningsindikator

**Comment by egonw** [comment]
I guess at some point ... we might want a l10n-ed version that much l10n-ed that it would country specific UI too :)

**Comment by fnielsen** [comment]
A tooltip on the column header could be nice.

**Comment by fnielsen** [comment]
There is also a Norwegian list.
